lib/bundler/ui/shell.rb

```diff
@@ -80,6 +81,9 @@ def silence
 
       # valimism
       def tell_me(msg, color = nil, newline = nil)
+        return if @told_messages.include? msg
```

Comment:
Yeah, the question is whether is desirable to show a message more than once to the user. When intended, the message can be add twice in a row, in the same string. Right?